vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -1,101 +1,70 @@
 
-        -   **Page for on-the-fly publishing**
+![configureContentType](./img/content_types2.png)
 
-        -   **Default content template**
+**4. Click the `Add` button.**
 
-        -   **Default content template for lists**
+This initializes the configuration of the selected Attribute Type. Each Attribute Type requires its own configuration. 
 
-*Default content template* and *Default content template for lists* are used
-respectively to show the detailed view of the Content and its
-representation as an element in a list.
+### Attribute Configuration
 
--   **Attributes**
+Adding an Attribute Type will load the "Add attribute" page, where `Type` is pre-filled with your selection. Follow the steps below to configure the Attribute.
 
-In the Attributes section, select the **Type** from the drop-down list.
-The list contains all the available attributes types that define which
-properties/attributes will characterize the Content.
+**1. Define the characteristics of the Attribute:**
 
-![image](./img/ContentTypes4.png)
+![configureAttributeType](./img/content_types3.png) 
 
-The Add button will start the configuration of the single attribute
-type. Every attribute type has its own configuration.
+- `Code`: It is mandatory to enter a unique name for the Attribute key (both Attributes of a 'Composite' Attribute Type may share a `Code` name). This field supports a string value (10 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. title).
 
-For example, to configure the **Text** Attribute Type, choose Text from
-the Types drop-down menu and then press the Add button.
+- `Name`: Enter a description of the Attribute. This field supports a string value (50 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. Title).
 
-![image](./img/ContentTypes5.png)
+- You may have the option to declare that the Attribute is `Mandatory`, `Searchable` and/or `Can be used as a filter in lists` via toggle buttons.
 
--   You should specify the following fields
+- You may have the option to choose an Attribute `Role`. Be sure to click the `Add` button after making your selection.  
 
--   **Type:** the value is pre-filled. It represents the selected
-    attribute type.
+- The remaining fields are not mandatory and may be left empty.
 
--   **Code**: You should insert the name of the attribute. (You
-    can insert max 10 characters uppercase and lowercase letters,
-    numbers, and special characters \_ ). (e.g. title)
+:::tip
+Certain Attribute Types allow you to assign the `Role` of `jacms:title - The main title of a Content`. This informs other plugins or services that the Attribute is a title, regardless of its key, or `Code`. Entering "title" for the `Code` avoids confusion when this `Role` is selected.
```

Comment:
other plugins or services >> can take out other, it makes it sound as if the subject is a plugin or service.